### ISSUE TITLE
Wrap inlined/long bibliography entries

### ISSUE THREAD

**Issue by frivoal** [tabatkins/bikeshed#2011]
Bikeshed prevents bibliography references from wrapping.
```css
[data-link-type=biblio] {
    white-space: pre;
}
```
 For `[[FOOBAR]]`, that's reasonable, but for `[[FOOBAR inline]]` or `[[FOOBAR|some decently long replacement]]` that's a lot less desirable.

Should that be fixed by adding a `class=has-replacement-text` (or similar) to such elements, and changing the selector to `[data-link-type=biblio]:not(.has-replacement-text)`, or should we not mark such things as `data-link-type=biblio`, or something else?

**Comment by tabatkins** [comment]
Yes, we should def distinguish between those cases; I had forgotten I had that CSS in place.

(It was there to prevent the `[]` from wrapping separately from the term, but any replacement text means it's just a normal link with text, so we should definitely not apply it there.)
